.github/workflows/spack-test.yaml

```diff
@@ -0,0 +1,51 @@
+# SPDX-FileCopyrightText: (C) The kokkos-fft development team, see COPYRIGHT.md file
+#
+# SPDX-License-Identifier: MIT OR Apache-2.0 WITH LLVM-exception
+
+name: CI (spack)
+
+on:
+  pull_request:
+    branches: [ "main" ]
+  
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        backend:
+          - name: nvcc
+            image: gcc
+            kokkos: +cuda +wrapper cuda_arch=90
+            kokkos_fft: device_backend=cufft
+          #- name: rocm
+          #  image: gcc
+          #  kokkos: +rocm amdgpu_target=gfx90a
+          #  kokkos_fft: hipfft
+
+    steps:
+      - name: Free Disk Space (Ubuntu)
+        uses: jlumbroso/free-disk-space@54081f138730dfa15788a46383842cd2f914a1be  # v1.3.1
+        with:
+          tool-cache: true
+          large-packages: false
+
+      - name: Checkout built branch
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # v4.2.2
+
+      - name: Clone and install spack
+        run: |
+          docker run -v ${{ github.workspace }}:/work ghcr.io/kokkos/kokkos-fft/base_${{ matrix.backend.image }}:latest \
+          bash -c "which python3 && git clone --depth=2 https://github.com/spack/spack.git && \
+                   spack/bin/spack install libelf"
```

Comment:
Why do we need to install libelf separately ?

Reply:
I just followed the spack docs. We do not need it?

Reply:
[Here](https://spack.readthedocs.io/en/latest/) I found

> Get Spack from the [GitHub repository](https://github.com/spack/spack) and install your first package:
> 
> git clone --depth=2 https://github.com/spack/spack.git
> cd spack/bin
> ./spack install libelf
> 
> Note
> 
> --depth=2 prunes the git history to reduce the size of the Spack installation.

Reply:
Ok I think it was only for illustration, no need to install it.